vault-operator/test/e2e/backup_restore_test.go

```diff
@@ -0,0 +1,238 @@
+// Copyright 2018 The vault-operator Authors
```

Comment:
I don't see the point to include the `backup_restore_test.go`. We need to have S3 for this to work. I am hoping that the test example we have shown here can be run locally. I believe that's what user expect as well.

cc/ @hasbro17

Reply:
@AlexNPavel We can remove backup_restore_test.go. The other basic and scale tests are good enough for example purposes. This test is just getting into the specifics of the vault-operator's reliance on the etcd-operator.